test/compare.numbers.test.js

```diff
@@ -0,0 +1,16 @@
+import compareNumbers from '../src/compare.numbers.js';
+
+const test = QUnit.test;
+
+test('decides if number is above or below 7', (assert) => {
+// Arrange
+    const input = 7;
+    const correct = 7;
```

Comment:
make sure to test for the other two outcomes that compareNumbers can give you. Also, rename that file to compareNumbers.js